app/controllers/api/v0/source_types_controller.rb

```diff
@@ -12,11 +12,11 @@ def create
       private
 
       def create_params
-        body_params.permit(:name, :product_name, :vendor)
+        body_params.permit(:name, :product_name, :vendor, :schema)
       end
 
       def list_params
-        params.permit(:name, :product_name, :vendor)
+        params.permit(:name, :product_name, :vendor, :schema)
```

Comment:
I don't think we'll need `:schema` here.  This is the params that can be passed to `model.where`, and I don't think people will search / filter by `:schema`.  I should probably come up with a better name for this method.  :smile:

Reply:
fixing it...